cluster.go

```diff
@@ -97,15 +102,30 @@ func NewCluster(
 		logger.Infof("IPFS Cluster v%s listening on:\n%s\n", Version, listenAddrs)
 	}
 
+	// Note, we already loaded peers from peerstore into the host
+	// in daemon.go.
 	peerManager := pstoremgr.New(host, cfg.GetPeerstorePath())
 
-	ctx, cancel := context.WithCancel(context.Background())
+	idht, err := dht.New(ctx, host)
```

Comment:
Is this a new cluster DHT or a new connection to the IPFS public DHT?

Reply:
cluster DHT